src/cmd/manage_nsfs.js

```diff
@@ -405,7 +408,8 @@ async function fetch_account_data(argv, config_root, from_file) {
                 distinguished_name: argv.user,
                 uid: !argv.user && argv.uid,
                 gid: !argv.user && argv.gid,
-                new_buckets_path: argv.new_buckets_path
+                new_buckets_path: argv.new_buckets_path,
+                fs_backend: argv.fs_backend
```

Comment:
if the user set "NSFS_NC_CONFIG_DIR_BACKEND": "GPFS" in config env, do we need to consider that here?

Reply:
actually it's 2 different backends, this fs_backend is of the storage_path and NSFS_NC_CONFIG_DIR_BACKEND is the fs_backend of the config_directory. but this reminded me that we take config_root_backend as an argument so I added NSFS_NC_CONFIG_DIR_BACKEND as its default, thanks!!